vcl_templates/www.vcl.erb

```diff
@@ -40,6 +40,55 @@ backend F_origin {
 <% end -%>
 }
 
+<%- if config['replatforming_traffic_split'] %>
+backend F_eks_origin {
```

Comment:
We may have an issue with the number of backends (5 is the default limit if I remember correctly)  but we shall see. Ref: https://docs.fastly.com/en/guides/resource-limits#service-domain-and-origin-limits

Reply:
I think this should bring us up to the 5 backend limit (as long as directors don't count as backends, which they shouldn't)

Reply:
Can we safely test this on the prod service on Tuesday (tomorrow) perhaps? Or are we confident that staging has the same number of existing backends so we're ok?